sourcegraph/about
# Update HubSpot forms

Fixing issue where HubSpot forms are loading twice.

Changes:
- Double HubSpot forms are now single forms with an additional CTA for `/demo`
- ChiliPiper has been moved to a separate hook and the prop removed from the HubSpot hook.

## To Test
- Check `/code-insights`, `/accelerate-dev-onboarding`, and `/fixing-vulnerabilities` and ensure nothing looks amiss.
